app/views/ile_query_mucr_response.scala.html

```diff
@@ -14,73 +14,91 @@
  * limitations under the License.
  *@
 
-@import java.time.format.DateTimeFormatter
-@import models.viewmodels.notificationspage.NotificationsPageSingleElement
-@import views.IleCodeMapper
-@import views.Title
-@import views.Title.NO_SECTION
-@import views.ViewDates
-@import models.notifications.queries.MucrInfo
+@import components.gds.gds_main_template
+@import components.ilequery.{response_buttons, response_parent, response_previous_movements, response_associated_consignments}
+@import models.notifications.queries.{MucrInfo, UcrInfo}
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.components.BackButton
+@import views.{IleCodeMapper, Title, TransportMapper}
 
-@this(main_template: views.html.templates.main_template)
+@this(
+        ileCodeMapper: IleCodeMapper,
+        transportMapper: TransportMapper,
+        govukLayout: gds_main_template,
+        parentSection: response_parent,
+        previousMovementsSection: response_previous_movements,
+        associatedConsignmentsSection: response_associated_consignments,
+        responseButtons: response_buttons,
+        govukButton: GovukButton,
+        govukSummaryList : GovukSummaryList,
+        govukTable: GovukTable,
+        govukTag: GovukTag
+)
 
-@(info: MucrInfo)(implicit request: Request[_], messages: Messages)
+@(info: MucrInfo, parentMucr: Option[MucrInfo], associatedConsignments: Seq[UcrInfo])(implicit request: Request[_], messages: Messages)
 
+@govukLayout(
+    useCustomContentWidth = true,
+    title = Title("ileQueryResponse.mucr.title"),
+    backButton = Some(BackButton(messages("site.back"), controllers.ileQuery.routes.IleQueryController.displayQueryForm()))) {
 
+    <div class="govuk-grid-row">
 
-@main_template(title = Title("ileQueryResponse.mucr.title")) {
-
-  @components.back_link(controllers.ileQuery.routes.IleQueryController.displayQueryForm())
-
-    @components.page_title(
-        Some(messages("ileQueryResponse.mucr.title")),
-        Some(info.ucr)
-    )
+        <div class="govuk-grid-column-two-thirds">
```

Comment:
Why do you use grid definitions here? `govukLayout` already defines `tow-thirds` width for the page content.
Should you need a different width you can use `govukFlexibleLayout`

Reply:
The tables need to be full width.  I copied the example of "movements" page in external service.

Reply:
Yes, you're right. I forgot how it was working.